diff --git a/node_modules/react-native/Libraries/NewAppScreen/components/DebugInstructions.js b/node_modules/react-native/Libraries/NewAppScreen/components/DebugInstructions.js
index 35bb4a8..205d1ad 100644
--- a/node_modules/react-native/Libraries/NewAppScreen/components/DebugInstructions.js
+++ b/node_modules/react-native/Libraries/NewAppScreen/components/DebugInstructions.js
@@ -8,7 +8,6 @@
  * @format
  */
 
-import type {Node} from 'react';
 import {Platform, StyleSheet, Text} from 'react-native';
 import React from 'react';
 
@@ -18,7 +17,7 @@ const styles = StyleSheet.create({
   },
 });
 
-const DebugInstructions: () => Node = Platform.select({
+const DebugInstructions = Platform.select({
   ios: () => (
     <Text>
       Press <Text style={styles.highlight}>Cmd + D</Text> in the simulator or{' '}
diff --git a/node_modules/react-native/Libraries/NewAppScreen/components/Header.js b/node_modules/react-native/Libraries/NewAppScreen/components/Header.js
index 4907466..01b58c4 100644
--- a/node_modules/react-native/Libraries/NewAppScreen/components/Header.js
+++ b/node_modules/react-native/Libraries/NewAppScreen/components/Header.js
@@ -8,13 +8,12 @@
  * @format
  */
 
-import type {Node} from 'react';
 import {ImageBackground, StyleSheet, Text, useColorScheme} from 'react-native';
 import React from 'react';
 import Colors from './Colors';
 import HermesBadge from './HermesBadge';
 
-const Header = (): Node => {
+const Header = () => {
   const isDarkMode = useColorScheme() === 'dark';
   return (
     <ImageBackground
diff --git a/node_modules/react-native/Libraries/NewAppScreen/components/HermesBadge.js b/node_modules/react-native/Libraries/NewAppScreen/components/HermesBadge.js
index 9ead0ca..993fa1d 100644
--- a/node_modules/react-native/Libraries/NewAppScreen/components/HermesBadge.js
+++ b/node_modules/react-native/Libraries/NewAppScreen/components/HermesBadge.js
@@ -9,11 +9,10 @@
  */
 
 import React from 'react';
-import type {Node} from 'react';
 import {StyleSheet, Text, useColorScheme, View} from 'react-native';
 import Colors from './Colors';
 
-const HermesBadge = (): Node => {
+const HermesBadge = () => {
   const isDarkMode = useColorScheme() === 'dark';
   const version =
     global.HermesInternal?.getRuntimeProperties?.()['OSS Release Version'] ??
diff --git a/node_modules/react-native/Libraries/NewAppScreen/components/LearnMoreLinks.js b/node_modules/react-native/Libraries/NewAppScreen/components/LearnMoreLinks.js
index b2d170e..71a34d9 100644
--- a/node_modules/react-native/Libraries/NewAppScreen/components/LearnMoreLinks.js
+++ b/node_modules/react-native/Libraries/NewAppScreen/components/LearnMoreLinks.js
@@ -9,8 +9,8 @@
  */
 
 import Colors from './Colors';
-import type {Node} from 'react';
-import openURLInBrowser from 'react-native/Libraries/Core/Devtools/openURLInBrowser';
+import {Linking} from 'react-native';
+let openURLInBrowser = (url) => Linking.openURL(url);
 import {
   StyleSheet,
   Text,
@@ -75,7 +75,7 @@ const links = [
   },
 ];
 
-const LinkList = (): Node => {
+const LinkList = () => {
   const isDarkMode = useColorScheme() === 'dark';
   return (
     <View style={styles.container}>
diff --git a/node_modules/react-native/Libraries/NewAppScreen/components/ReloadInstructions.js b/node_modules/react-native/Libraries/NewAppScreen/components/ReloadInstructions.js
index 0294113..ca5b9a3 100644
--- a/node_modules/react-native/Libraries/NewAppScreen/components/ReloadInstructions.js
+++ b/node_modules/react-native/Libraries/NewAppScreen/components/ReloadInstructions.js
@@ -8,7 +8,6 @@
  * @format
  */
 
-import type {Node} from 'react';
 import {Platform, StyleSheet, Text} from 'react-native';
 import React from 'react';
 
@@ -18,7 +17,7 @@ const styles = StyleSheet.create({
   },
 });
 
-const ReloadInstructions: () => Node = Platform.select({
+const ReloadInstructions = Platform.select({
   ios: () => (
     <Text>
       Press <Text style={styles.highlight}>Cmd + R</Text> in the simulator to
